src/webaudio.js

```diff
@@ -656,7 +656,7 @@ export default class WebAudio extends util.Observer {
 
         this.scheduledPause = end;
 
-        this.source.start(0, start, end - start);
+        this.source.start(0, start);
```

Comment:
this doesn't seem right and break existing stuff..

Reply:
this doesn't break anything; the playback is still stopped at `end` here: https://github.com/katspaugh/wavesurfer.js/blob/2eb72e06b60b2e8ee05f97c1674f187dcaa4fea8/src/webaudio.js#L254